scrapers/AniDB.yml

```diff
@@ -0,0 +1,137 @@
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+xPathScrapers:
+  performerSearch:
+    performer:
+      Name: //td[@class="relid"]/a
+      URL:
+        selector: //td[@class="relid"]/a/@href
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://anidb.net
+  performerScraper:
+    common:
+      $info: //div[@class="g_section info"]
+      $tab_1_pane: //div[@class="g_section info"]//div[@id="tab_1_pane"]
+      $looks: //div[@class="g_section info"]//div[@id="tab_1_pane"]//tr[contains(@class, 'looks')]
+    performer:
+      Name: $tab_1_pane//tr[contains(@class, 'mainname')]//span[@itemprop="name"]
+      Aliases: $tab_1_pane//tr[contains(@class, 'official')]//label[@itemprop="alternateName"]
+      Gender: $tab_1_pane//tr[contains(@class, 'gender')]//span[@itemprop="gender"]
+      Ethnicity: $tab_1_pane//tr[contains(@class, 'entity')]//span[@class="tagname"]
+      HairColor: $looks//span[contains(@class, 'tagname') and contains(text(), 'hair')]
+      EyeColor: $looks//span[contains(@class, 'tagname') and contains(text(), 'eyes')]
+      Height: $tab_1_pane//tr[contains(@class, 'height')]//span[@itemprop="height"]
+      Weight: $tab_1_pane//tr[contains(@class, 'weight')]//span[@itemprop="weight"]
+      #Measurements: Todo
+      URL: //link[@rel="canonical"]/@href
+      Details: 
+        selector: //div[@itemprop="description"]//text()
+        concat: "\n"
+      Tags:
+        Name: $tab_1_pane//span[@class="g_tag"]//span[@class="tagname"]
+      Image: $info//div[@class="image"]//img/@src
+
+  sceneSearch:
+    scene:
+      Title: //td[@class="relid"]/a
+      URL:
+        selector: //td[@class="relid"]/a/@href
+        postProcess:
+          - replace:
+              - regex: ^
+                with: https://anidb.net
+      Image: //td[@class="thumb anime"]//img/@src
+  sceneScraper:
+    common:
+      $info: //div[@class="g_section info"]
+    scene:
+      Title: $info//div[@id="tab_1_pane"]//span[@itemprop="name"]
+      Details: 
+        selector: //div[@itemprop="description"]//text()
+        concat: "\n"
+      Tags:
+        Name: $info//div[@id="tab_1_pane"]//span[@class="tagname"]
+      Performers:
+        Name: //div[@id="characterlist"]//div[contains(@class, 'main character') or contains(@class, 'secondary cast')]//div[@itemprop="character"]/a/span
+      Studio:
+        Name: $info//table[@id="staffoverview"]//tr[last()]/td[@class="name creator"]/a
+      Image: $info//div[@class="image"]//img/@src
+      URL: //link[@rel="canonical"]/@href
+      
+driver:
+  cookies:
+    - CookieURL: "https://anidb.net/"
+      Cookies:
+        # Access adult content requires a anidb account
+        # Replace value field
+        - Name: "adbsess"
+          Domain: "anidb.net"
+          Value: "" # Enter the value of the 'adbuin' here

```

Comment:
I assume the comment here should be the value of 'adbsess' ?

Reply:
yes, my mistake